### PR TITLE
update snippets: integrate-openvino-with-your-application#step-4-set-…

### PR DESCRIPTION
**Pull request by valord577**
Just match context. (refs https://github.com/openvinotoolkit/openvino/pull/25532#discussion_r1697037985)

Screenshot 2024-07-30 at 22 06 43
Screenshot 2024-07-30 at 22 08 41

### CODE REVIEW

**Comment by valord577** [comment]
@vladimir-paramuzov Could you please take a time to review this PR?

File: docs/snippets/src/main.c [hunk 1]
```diff
@@ -61,7 +61,7 @@ void * memory_ptr = NULL;
 //! [part4]
 // Get input port for model with one input
 ov_output_const_port_t* input_port = NULL;
-ov_model_const_input(model, &input_port);
+ov_compiled_model_input(compiled_model, &input_port);
```
Comment (riverlijunjie) [inline docs/snippets/src/main.c:64]: Is there any reason to change to be `ov_compiled_model_input`? I supposed they should get the same result, right?

Comment (valord577) [reply docs/snippets/src/main.c:64]: @riverlijunjie Thanks for code review! For your question, just match context.

Screenshot 2024-07-30 at 22 06 43
Screenshot 2024-07-30 at 22 08 41

Comment (riverlijunjie) [reply docs/snippets/src/main.c:64]: It makes sense, thank your clarification!

**Review by riverlijunjie — APPROVED** [review]

File: docs/snippets/src/main.c (hunk 1 above)
Comment (riverlijunjie) [reply docs/snippets/src/main.c:64]: Please update description for this PR, thanks!

Comment (valord577) [reply docs/snippets/src/main.c:64]: > Please update description for this PR, thanks!

Done.

**Comment by valord577** [comment]
@riverlijunjie Could you please help with the CI issue?

**Comment by riverlijunjie** [comment]
> @riverlijunjie Could you please help with the CI issue?

rebuilt

**Comment by ilya-lavrenov** [comment]
build_jenkins

**Pull request merged**